public_suffix_list.dat

```diff
@@ -10074,9 +10074,6 @@ trading
 // training : 2013-11-07 Binky Moon, LLC
 training
 
-// travel :  Dog Beach, LLC
-travel
```

Comment:
@weppos @dnsguru Do you know what's up here? IANA still shows .travel delegated ( https://www.iana.org/domains/root/db/travel.html ), but I'm not seeing it in the CSV ( https://newgtlds.icann.org/newgtlds.csv )

Reply:
I actually found this issue long time ago (sigh, it was July 2018) when I run this script, and I always meant to follow up :(

I also wrote to Francisco at ICANN, and got back:

> Dear Simone Carletti,
> 
> Thank you for your time and patience. 
> 
> .TRAVEL is not expected to be on the New gTLD Registry Operator CSV Report because it is not a new gTLD.
> 
> ICANN has published a new report in JSON format that you may find useful, which lists all gTLDs and some of their characteristics: https://www.icann.org/resources/registries/gtlds/v2/gtlds.json
> 
> The existing New gTLDs CSV report has also been relocated here: https://www.icann.org/resources/registries/gtlds/v1/newgtlds.csv
> 
> Please note that the old report location (https://newgtlds.icann.org/newgtlds.csv) is still available, but it is planned to be replaced with a HTTP 301 response in the following weeks in order to redirect requests to the new report location.
> 
> I hope this information has been of assistance.
> 
> I will now close this case. If you have any further questions or concerns, please feel free to contact us.

@dnsguru also confirmed it to me, in a private conversation.

---

I'm not sure how it ended up in this list. Perhaps at some point ICANN returned it as newGTLd in their feed.

We probably want to move it to the ICANN section outside the newGTLd.

Reply:
Thanks. I opened https://github.com/publicsuffix/list/issues/831